docs/pages/_meta.json

```
@@ -1,4 +1,8 @@
 {
   "index": "Introduction",
-  "hooks": "NFT Hooks"
+  "graphql-api": "GraphQL API",
+  "hooks": "NFT Hooks",
```

We could just name it hooks:
```suggestion
  "hooks": "Hooks",
```

---

Maybe even better `React Hooks`, maybe hooks is a bit too generic

---

done